gameplay/avalonRoom.js

```diff
@@ -94,6 +94,7 @@ module.exports = function (host_, roomId_, io_, maxNumPlayers_, newRoomPassword_
 	
 	this.proposedTeam = [];
 	this.lastProposedTeam = [];
+	this.missionTeam = [];
```

Comment:
I think you don't need this. You should be able to get mission players from the vote history data coming into the client side? Might be a bit tricky but the data is already there.

Reply:
I get that we don't want to add more properties if we don't have to, but I've been trying to figure out how to get the members of the team from the vote history data and it's a little difficult because of the way the vote history is set up. Also, the missions that are voted down are included in the vote history but not marked as failed, and I don't see an easy way to filter those out without tallying up all the votes every time the user hovers over the mission table.  Would it not be cleaner to keep the missionTeam setup I created?